packages/gatsby-theme-carbon/src/components/ResourceCard/ResourceCard.js

```diff
@@ -13,100 +13,140 @@ import { settings } from 'carbon-components';
+    <>
+      {type === 'mini' ? (
+        <div className={`${prefix}--resource-card__mini`}>
+          <div className={`${prefix}--resource-card__mini-content-wrapper`}>
+            <div className={`${prefix}--resource-card__mini-title`}>
+              {title}
+            </div>
+            {children === undefined ? (
+              <div style={{ height: '20px' }}>
+                {actionIcon === 'launch' && !disabled ? (
+                  <Launch20 aria-label="Open resource" />
+                ) : null}
+                {actionIcon === 'arrowRight' && !disabled ? (
+                  <ArrowRight20 aria-label="Open resource" />
+                ) : null}
+                {actionIcon === 'download' && !disabled ? (
+                  <Download20 aria-label="Download" />
+                ) : null}
+                {actionIcon === 'email' && !disabled ? (
+                  <Email20 aria-label="Email" />
+                ) : null}
+                {actionIcon === 'disabled' || disabled === true ? (
+                  <Error20 aria-label="disabled" />
+                ) : null}
+              </div>
+            ) : (
+              <div className={`${prefix}--resource-card__mini-icon--img`}>
+                {children}
+              </div>
+            )}
           </div>
         </div>
-      </div>
+      ) : (
+        <>
+          {subTitle && (
+            <h5 className={`${prefix}--resource-card__subtitle`}>{subTitle}</h5>
+          )}
+          {title && (
+            <h4 className={`${prefix}--resource-card__title`}>{title}</h4>
+          )}
+          <div className={`${prefix}--resource-card__icon--img`}>
+            {children}
+          </div>
+          <div className={`${prefix}--resource-card__icon--action`}>
+            {actionIcon === 'launch' && !disabled ? (
+              <Launch20 aria-label="Open resource" />
+            ) : null}
+            {actionIcon === 'arrowRight' && !disabled ? (
+              <ArrowRight20 aria-label="Open resource" />
+            ) : null}
+            {actionIcon === 'download' && !disabled ? (
+              <Download20 aria-label="Download" />
+            ) : null}
+            {actionIcon === 'email' && !disabled ? (
+              <Email20 aria-label="Email" />
+            ) : null}
+            {actionIcon === 'disabled' || disabled === true ? (
+              <Error20 aria-label="disabled" />
+            ) : null}
+          </div>
+        </>
+      )}
+    </>
+  );
+
+  let cardContainer;
+  if (disabled === true || href === undefined) {
+    cardContainer = <div className={carbonTileclassNames}>{cardContent}</div>;
+  } else if (isLink === true) {
+    cardContainer = (
+      <Link to={href} className={carbonTileclassNames} {...rest}>
+        {cardContent}
+      </Link>
+    );
+  } else {
+    cardContainer = (
+      <a href={href} className={carbonTileclassNames} {...rest}>
+        {cardContent}
+      </a>
     );
   }
-}
+
+  return (
+    <>
```

Comment:
A lot of these fragments are unnecessary, see which one's you can remove and get away with it.

You'll should only need them very rarely, when you need two adjacent components that don't make sense to have wrapped in some container.

Reply:
When I remove them, the page won't won't render. Do i just use divs instead?

Reply:
figure it out. I'll show you tomorrow :)